wrappers/mity/call/wrapper.py

```diff
@@ -0,0 +1,15 @@
+from snakemake.shell import shell
+import os
+
+log = snakemake.log_fmt_shell(stdout=True, stderr=True)
+
+family = snakemake.wildcards.family
+outdir = snakemake.params.outdir
+prefix = snakemake.params.prefix
+tool = snakemake.params.tool
+print(snakemake.input.bam)
+
+mod1= (" module load python; ")
+mod2= (" module load freebayes/1.3.1; ")
```

Comment:
Why do we need to load python and freebayes modules if they are present in the environment.yaml?

Reply:
Note: Discussed and agreed to keep this the way it is for now.